.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
For latest mvnw, we don't need this Java class. For his older version, I am not sure whether it is proper to add ASF header.